cephfs/admin/volume.go

```diff
@@ -20,6 +21,34 @@ func (fsa *FSAdmin) ListVolumes() ([]string, error) {
 	return parseListNames(res)
 }
 
+// FileSysPool contains the name of a file system as well as the metadata and
+// data pools. Pool information is available by ID or by name.
+type FileSysPool struct {
```

Comment:
Maybe call this  `FSPool`, similar to `FSAdmin`?

Reply:
Haha. That's what I started out with, and then was concerned it seemed *too* much like FSAdmin. How about FSPoolInfo to help indicate it is just reporting on the association between file system and pools (rather than managing them)?

Reply:
Yeah, FSPoolInfo sounds good too. Thanks!

Reply:
Updated.